agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/AffectedGenomicModelIndexer.java

```diff
@@ -21,6 +21,8 @@ public class AffectedGenomicModelIndexer extends Indexer {
 	private final HashMap<String, Object> params = new HashMap<>() {{
 		put("internal", false);
 		put("obsolete", false);
+		//put("primaryExternalId", "ZFIN:ZDB-FISH-181119-4");
```

Comment:
Please remove the comments

Reply:
@abecerra generally we ask for commented out code to be removed or unused code to be removed, in this case adding params and commenting them out actually helps with development whereas the other types hinder development.